pkg/parser/validate/parameters.go

```diff
@@ -73,7 +73,19 @@ func (val Validate) checkParamSimpleType(param ast.ParameterValue, stepName stri
 			val.createParameterError(param, stepName, definedParam.GetType())
 		}
 		for _, value := range values {
-			if value.Type != "steps" {
+			if value.Type == "string" {
+				commandName := value.Value.(string)
+				_, commandExists := val.Doc.Commands[commandName]
```

Comment:
should work with jobs too

you can use `val.Doc.DoesJobExist()` and `val.Doc.DoesCommandExist()`